shared/settings/index.js

```diff
@@ -6,14 +6,20 @@ import {routeAppend} from '../actions/router'
 import about from './about'
 import logSend from '../dev/log-send'
 import account from './account'
-import billing from './about'
-import appPrefs from './about'
-import invites from './about'
-import notifs from './about'
-import deleteMe from './about'
+// import billing from './about'
+// import appPrefs from './about'
+// import invites from './about'
+// import notifs from './about'
+// import deleteMe from './about'
 import devMenu from '../dev/dev-menu'
 import flags from '../util/feature-flags'
 
+const billing = about
```

Comment:
this is all fake code anyways, but we shouldn't be doing this import so lets just keep the same reference